### ISSUE TITLE
Warning when VCalendar or VDatePicker is mounted

### ISSUE THREAD

**Issue by acordy** [samk-dev/nuxt-vcalendar#15]
Hi, first off, thanks for the module!

I'm receiving these console warnings when either component (VCalendar or VDatePicker) is mounted:

> [Vue warn] Computed is still dirty after getter evaluation, likely because a computed is mutating its own dependency in its getter. State mutations in computed getters should be avoided.

With VCalendar, this warning appears six times on mount, and with VDatePicker it appears four times. I'm not using SSR, and I've tried wrapping it in ClientOnly, though that should not affect anything in client-side rendering.

I'm not sure if this is an issue with VCalendar itself, or the module, or if it's just me. Are you seeing these warnings as well?

**Comment by samk-dev** [comment]
Hi @acordy sorry for the late reply I have a day job 😅. Thanks for reporting

I am using the module on various projects with different versions of Nuxt and Vue and works good if wrapped in ClientOnly during SSR.

If you can share a reproduction I can take a look at it with you.

Regards

**Comment by acordy** [comment]
Hi @samk-dev sorry myself for the half-year-later reply! We finally got around to having a look at it again this week, and in a minimal setup we were not able to reproduce the issue. In the end it was related to a series of factors unique to our environment, and we were able to resolve it.

So, sorry for the false alarm, there was nothing wrong with your code. And let me take the opportunity to thank you for a great component!
